domain_scoring/domain_value_transformer.py

```diff
@@ -0,0 +1,108 @@
+from typing import List, Tuple, Dict
+from numpy import arange
+from util.datastructures import MetaPath
+
+SMALLER = 0
+LARGER = 1
+
+class DomainValueTransformer:
+    """
+    Abstract class. Domain Value Transformers are strategies that can be used to transform meta-path ordering predictions
+    into an actual numeric domain value.
+    """
+
+    def transform(self, metapaths_pairs: List[Tuple[MetaPath, MetaPath]], classification: List[int]) -> List[
+        Tuple[MetaPath, float]]:
```

Comment:
maybe use the [type variables](https://docs.python.org/3/library/typing.html#type-aliases) from the typing package to make this more readable eg. ` List[Tuple[MetaPath, float]]` to ` List[Rating]`

Reply:
Originally `MetaPathRating` from `datastructures.py` was intended as this rating object. We could also use this.

Reply:
Well, the point is that this is exactly a tuple of two meta-paths and not a rating.